### PR TITLE
WearOS version

### PR DESCRIPTION
**Pull request by rezazarchi**
related to issue #176

### CODE REVIEW

**Comment by rezazarchi** [comment]
Hey @Kaaveh 
I've added a new Android application module for Wear OS! I've included all the necessary dependencies and set up a basic MainActivity (it's empty for now).

I'd love to hear your thoughts on this initial setup. Please let me know if you have any feedback. Thanks!

**Review by Kaaveh — DISMISSED** [review]

**Comment by Kaaveh** [comment]
Hi @rezazarchi 
It's amazing! Please continue.

**Comment by rezazarchi** [comment]
I changed modules structure to remove all WearOS-related dependencies from the current modules. We now have separated design, navigation, and UI modules for the WearOS version.

I added the remaining implementations for showing the list and details. However, there's still some work to do:

- `QuadLineChart` still needs optimization for Wear UI. I couldn't resolve this issue yet.
- We might improve the navigation to the details screen. I used `navController.navigate(***)` for this, but your previous solution for the details screen in the app module was slightly different.
- The favorite feature hasn't been implemented yet.
- Swipe functionality for each item hasn't been implemented yet.
- The module structure might need review and refactoring if necessary.
- Unit and UI tests haven't been implemented yet.

**Review by Kaaveh — DISMISSED** [review]
Excellent!

File: app-watch/app/proguard-rules.pro [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (Kaaveh) [inline app-watch/app/proguard-rules.pro:1]: Can this file be deleted?

File: app-watch/navigation/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Kaaveh) [inline app-watch/navigation/src/main/AndroidManifest.xml:1]: I think this file can be deleted.

File: app-watch/ui/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Kaaveh) [inline app-watch/ui/src/main/AndroidManifest.xml:1]: I think this file can be deleted.

File: gradle/libs.versions.toml [hunk 4]
```diff
@@ -2,6 +2,7 @@
 projectApplicationId = "ir.composenews"
 projectVersionName = "1.0.0"
 projectMinSdkVersion = "23"
```
Comment (Kaaveh) [inline gradle/libs.versions.toml:4]: I guess the build issue come from here! 🤔

**Review by Kaaveh — CHANGES_REQUESTED** [review]

**Review by Kaaveh — DISMISSED** [review]

**Review by Kaaveh — APPROVED** [review]

**Pull request merged**